wbooks/src/app/screens/BookList/index.js

```diff
@@ -13,11 +14,16 @@ import BOOKS from './books';
 class BookList extends Component {
 
 	renderItem = ({ item }) => (
-		<BookItem 
-			title= {item.title}
-			author= {item.author}
-			uri= {item.image_url}
-		/>
+		<TouchableOpacity onPress={
+			() => this.props.navigation.navigate('Details', {
```

Comment:
What do you think about this @guidoprinc ? I don't like it.

Reply:
he is passing item to the function, there is no way to avoid this. Maybe in just one line will be better, but there will be an arrow function anyway